tests/testthat/test-install.R

```diff
@@ -275,15 +274,11 @@ test_that("install() installs inferred dependencies", {
   skip_on_cran()
   renv_tests_scope("breakfast")
 
-  # use dummy library path
-  templib <- renv_scope_tempfile("renv-library-")
-  ensure_directory(templib)
-  renv_scope_libpaths(templib)
-
   # try installing packages
-  install()
+  records <- install()
 
   # validate that we've installed breakfast + deps
+  expect_length(records, 2L)
```

Comment:
I think this should be 4, since:

```
> tools:::package_dependencies("breakfast", recursive = TRUE)
$breakfast
[1] "oatmeal" "toast"   "bread"  
```

Reply:
And indeed it is 4 😄